omnigan/data.py

```diff
@@ -380,7 +380,7 @@ def get_loader(mode, domain, opts):
             mode, domain, opts, transform=transforms.Compose(get_transforms(opts))
         ),
         batch_size=opts.data.loaders.get("batch_size", 4),
-        shuffle=True,
+        shuffle=opts.data.loaders.shuffle,
```

Comment:
I think we should always shuffle that's why I had removed that option. Validation images logged in comet are not created from iterating over the val loaders, just stored once before training into the `display_images`